src/tests/run.py

```diff
@@ -875,8 +875,8 @@ def run_tests(args,
         os.environ["LargeVersionBubble"] = "true"
 
     if gc_stress:
-        print("Running GCStress, extending timeout to 120 minutes.")
-        per_test_timeout = 120*60*1000
+        print("Running GCStress, extending timeout to 240 minutes.")
```

Comment:
It seems it would make sense to apply the gc_stress timeout as the last one as it is by far the slowest mode. And maybe jut multiply the previously picked timeout by a constant instead of setting a fixed timeout. I guess that any "type" of test would become somehow proportionally slower with GC stress enabled.

Reply:
Thanks for the suggestion, addressed in 2nd commit, can you please take another look?